test/system/700-play.bats

```diff
@@ -1035,3 +1035,72 @@ spec:
     run_podman rm -f -t 0 -a
     run_podman rmi automount_test
 }
+
+@test "podman ply with multi automount volume" {
+    cat >$PODMAN_TMPDIR/Containerfile1 <<EOF
+FROM $IMAGE
+RUN mkdir /test1 /test_same
+RUN touch /test1/a /test1/b /test1/c
+RUN echo "I am from test1 image" > /test_same/hello_world
+VOLUME /test1
+VOLUME /test_same
+EOF
+
+    cat >$PODMAN_TMPDIR/Containerfile2 <<EOF
+FROM $IMAGE
+RUN mkdir /test2 /test_same
+RUN touch /test2/asdf /test2/ejgre /test2/lteghe
+RUN echo "I am from test2 image" > /test_same/hello_world
+VOLUME /test2
+VOLUME /test_same
+EOF
+
+    run_podman build -t test1 -f $PODMAN_TMPDIR/Containerfile1
+    run_podman build -t test2 -f $PODMAN_TMPDIR/Containerfile2
+
+    fname="/$PODMAN_TMPDIR/play_kube_wait_$(random_string 6).yaml"
+    cat >$fname <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: test
+  name: test_pod
+  annotations:
+    io.podman.annotations.kube.image.volumes.mount/testctr: test1;test2
+spec:
+  restartPolicy: Never
+  containers:
+    - name: testctr
+      image: $IMAGE
+      command:
+      - top
+    - name: testctr_not_mounted
+      image: $IMAGE
+      command:
+      - top
+EOF
+
+    run_podman kube play $fname
+
+    run_podman run --rm test1 ls /test1
+    run_out_test1="$output"
```

Comment:
Please check this for validity. Comparing the `ls` output is not useful if both show mistaken results.